frontend/src/fsrs_cards.tsx

```diff
@@ -328,11 +328,40 @@ const FSRSCards: React.FC<FSRSCardsProps> = ({
       handleFlip();
     }
   }, [currentCard, correctGuesses, typeInputValue, flipped]);
-
   const sortedGuesses = useMemo(() => {
     return Array.from(correctGuesses).sort((a, b) => a.localeCompare(b));
   }, [correctGuesses]);
 
+  const placeholder = useMemo(() => {
+    if (
+      !currentCard ||
+      !currentCard.alphagram ||
+      correctGuesses.size === 0 ||
+      flipped
+    ) {
+      return "Guess…";
+    }
+
+    const remainingWords =
+      currentCard.alphagram.words.length - correctGuesses.size;
+
+    return `${remainingWords} ${
```

Comment:
❌ self-review: need to respect the `Show number of words in cards` display setting